platform/src/core/apis/Collocation.js

```diff
@@ -1,8 +1,7 @@
-import axios from 'axios';
 import { COLLOCATION } from '../urls/deviceMonitoring';
-import { NEXT_PUBLIC_API_TOKEN } from '../../lib/envConstants';
+import createAxiosInstance from './axiosConfig';
 
 export const resetCollocationBatch = async (data, params) =>
-  await axios
-    .patch(`${COLLOCATION}/reset?token=${NEXT_PUBLIC_API_TOKEN}`, data, { params })
+  await createAxiosInstance(false)
```

Comment:
Hi @OchiengPaul442 , please shed more light on this `false`? Why is it a` false`?

Reply:
@Baalmart, Error on my part was testing with the token because of a 401 error on my side and forgot to remove the fales.